docker/debian-stable

```diff
@@ -31,34 +31,10 @@ RUN apt-get update \
 
 COPY docker/certs/ /usr/local/share/ca-certificates
 RUN update-ca-certificates
+RUN apt-get clean
```

Comment:
FWIW, this doesn't actually make the images smaller because it's adding new layers. If you want small images, either clean (`rm -rf /var/lib/apt/lists/*`) as part of the same `RUN` that does the installation or use multi-stage builds. For whatever reason, my builds never `apt-get clean` and yet `/var/cache/apt` is empty.

Reply:
Thanks. I will look to see if cleaning they way you describe noticeably reduces the size of the image. It might not given all the dependencies of installing Python.

Reply:
Cleaning `/var/lib/apt/lists` reduced the image size by only about 20 MB out of 750+ MB. Most of the space comes from stuff in `/usr/lib`.